source/reference/beta-program/destinationDataHandling.txt

```diff
@@ -0,0 +1,111 @@
+.. _c2c-beta-destination-data-handling:
+
+===========================================================
+Handle Destination Data with destinationDataHandling Option
+===========================================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. include:: /includes/destinationDataHandling-introduction.rst
+
+Syntax
+------
+
+To enable the ``destinationDataHandling`` feature in ``mongosync``:
+
+.. code-block:: shell
+
+   mongosync <options> --enableFeatures destinationDataHandling
+
+For the other ``mongosync`` options, see :ref:`c2c-mongosync`.
+
+To set a ``"destinationDataHandling"`` option string:
+
+.. code-block:: shell
+   :emphasize-lines: 5
+
+   curl <host>:<port>/api/v1/start -XPOST \
+   --data '
+      {
+         <options>,
+         "destinationDataHandling": <string>
+      } '
+
+Command Option
+--------------
+
+The following table shows the strings you can set for
+``"destinationDataHandling"``:
+
+.. list-table::
+   :header-rows: 1
+   :widths: 20 80
+ 
+   * - String
+     - Description
+ 
+   * - ``"dropAllData"``
```

Comment:
This actually isn't supported in the beta binary. We can use `"requireEmptyDestination"` though, which is also the default.